### ISSUE TITLE
MO4 uses deprecated sniff Generic.Functions.CallTimePassByReference

### ISSUE THREAD

**Issue by xalopp** [mayflower/mo4-coding-standard#218]
When I run `vendor/bin/phpcs -s --standard=MO4`, I get this warning message:

```
WARNING: The MO4 standard uses 1 deprecated sniff
-------------------------------------------------------------------------------
-  Generic.Functions.CallTimePassByReference
   This sniff has been deprecated since v3.12.1 and will be removed in v4.0.0.

Deprecated sniffs are still run, but will stop working at some point in the future.

ERROR: You must supply at least one file or directory to process.
```

**Comment by xalopp** [comment]
Relates to https://github.com/djoos/Symfony-coding-standard/pull/204

**Comment by mmoll** [comment]
> Relates to [djoos/Symfony-coding-standard#204](https://github.com/djoos/Symfony-coding-standard/pull/204)

I guess it will be fixed by the referenced upstream PR?
